src/main/java/org/aludratest/eclipse/vde/internal/editors/VisualDataEditorPage.java

```diff
@@ -139,6 +139,17 @@ public void widgetSelected(SelectionEvent e) {
 			}
 		});
 		tiIgnore.setEnabled(false);
+		
+		final ToolItem tiExternalTestId = new ToolItem(toolbar, SWT.CHECK);

```

Comment:
Sorry to discover this only now - SWT.CHECK seems wrong here, because that would be a toggle button. SWT.PUSH seems to be more correct here, doesn't it?